engine/build.gradle

```diff
@@ -71,18 +91,18 @@ configurations.all {
 // Primary dependencies definition
 dependencies {
     // White list packages {@see ExternalApiWhiteList class}
-    apiElements group: 'com.google.guava', name: 'guava', version: '23.0'
-    apiElements group: 'com.google.code.gson', name: 'gson', version: '2.6.2'
-    apiElements group: 'net.sf.trove4j', name: 'trove4j', version: '3.0.3'
-    apiElements group: 'org.lwjgl.lwjgl', name: 'lwjgl', version: LwjglVersion
-    apiElements(group: 'com.snowplowanalytics', name: 'snowplow-java-tracker', version: '0.8.0') {
+    api group: 'com.google.guava', name: 'guava', version: '23.0'
+    api group: 'com.google.code.gson', name: 'gson', version: '2.6.2'
+    api group: 'net.sf.trove4j', name: 'trove4j', version: '3.0.3'
+    api group: 'org.lwjgl.lwjgl', name: 'lwjgl', version: LwjglVersion
+    api(group: 'com.snowplowanalytics', name: 'snowplow-java-tracker', version: '0.9.0') {
         exclude group: 'org.slf4j', module: 'slf4j-simple'
     }
-    apiElements group: 'org.joml', name: 'joml', version: '1.9.24'
-    apiElements group: 'java3d', name: 'vecmath', version: '1.3.1'
+    api group: 'org.joml', name: 'joml', version: '1.9.24'
+    api group: 'java3d', name: 'vecmath', version: '1.3.1'
     // Light and Shadow use it
-    apiElements group: 'com.miglayout', name: 'miglayout-core', version: '5.0'
-    apiElements group: 'org.codehaus.plexus', name: 'plexus-utils', version: '1.5.6'
+    api group: 'com.miglayout', name: 'miglayout-core', version: '5.0'
+    api group: 'org.codehaus.plexus', name: 'plexus-utils', version: '1.5.6'
```

Comment:
you can reduce lines by fold `api` and `implementation`  by libs.
`api` contains `implementation` (checked on jupiter deps in `engine-tests`)

Reply:
Yep, DRY ! 
apiElements is deep technical configuration.. near gradle's internal.
I did't find it rather :)